addon-test-support/helpers.js

```diff
@@ -7,6 +7,9 @@ const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
 
+import config from 'dummy/config/environment';
```

Comment:
@rwjblue I had trouble importing the config from the consuming app, using the full path to the `dummy` config worked, but it's not right.